paddle/scripts/paddle_build.sh

```diff
@@ -50,6 +50,8 @@ function init() {
     if [ -z "${SCRIPT_NAME}" ]; then
         SCRIPT_NAME=$0
     fi
+
+    export MKL_DEBUG_CPU_TYPE=7 # Enable AVX512 instructions
```

Comment:
how about add this flag only for `fc_mkldnn` tests?

Reply:
Indeed, the flag should only for fc now, should not be added in `paddle_build.sh`.

I suppose the problem you proposed of `test_analyzer_seq_pool1` is caused by this global flag.